common/src/main/java/io/homeassistant/companion/android/database/wear/EntityStateComplicationsDao.kt

```diff
@@ -0,0 +1,24 @@
+package io.homeassistant.companion.android.database.wear
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+
+@Dao
+interface EntityStateComplicationsDao {
```

Comment:
This should be added to the `DatabaseModule.kt` file as well so it can be injected.